### ISSUE TITLE
New Task: Routinely pull new data from OTN as available

### ISSUE THREAD

**Issue by laurabrenskelle** [ioos/marine_life_data_network#74]
### Who is requesting this?

@ioos/marine-life

### What is being requested?

From the Trello card: "3.2.1 Trigger automated messaging system to routinely pull metadata, metadata updates, and matched detection data from OTN and keep a local cache for ATN curation.

Develop ingestion pipeline in packrat to parse ADR to routinely pull new data and metadata from OTN as available."

### Provide a description or any other important information.

This is from a Marine Life Data Network Trello card with a deadline of April 30, 2023.

**Comment by laurabrenskelle** [comment]
There is another MLDN Trello card that says: "3.6.1 Develop application to allow OTN products and search services to be consumed by ATN data portal. ATN data portal is harvesting into the ADR metadata (project, gear, tagging logs) rolled up from regional nodes from projects that are no longer under embargo." Are these tasks one in the same, or should they be split into two separate issues as they were on Trello? This Trello card had the dates May 1, 2023 - July 31, 2023.

**Comment by MathewBiddle** [comment]
This is dependent on:
* #51

**Comment by mmckinzie** [comment]
Yes, Trello cards are related but outdated and should be depreciated. The MLDN told Axiom to hold-off on any acoustic related tasking back in Nov/Dec 2023 until we can finalize the role(s) ATN should play in regard to supporting the needs of the acoustic telemetry community and how best to facilitate meeting this need.

**Comment by mmckinzie** [comment]
Moving forward it would likely be best to harvest metadata from the OTN geoserver and data from OBIS.

**Comment by MathewBiddle** [comment]
This should simply be moved to blocked, dependent on decisions made in https://github.com/ioos/marine_life_data_network/issues/51

**Comment by MathewBiddle** [comment]
Closing. Not relevant and needs evaluation as to how to integrate OTN metadata with ATN.
